### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Oznobishina**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -7,4 +7,19 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test('Should return not falsy: ', () => {
```
Comment (Ivan-Shvorak) [inline src/getCoinCombination.test.js:11]: We don't need this test. Other tests include this check.

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -7,4 +7,19 @@ describe('getCoinCombination', () => {
...
+    expect(getCoinCombination(10)).not.toBeFalsy();
+  });
+
+  test('Should return defined: ', () => {
+    expect(getCoinCombination(7)).toBeDefined();
+  });
+
+  test('Should return corect equal: ', () => {
```
Comment (Ivan-Shvorak) [inline src/getCoinCombination.test.js:19]: 1. corect => correct
2. Make them atomic, one test = one check
3. use boundary values and equivalence classes during test creation

**Review by Ivan-Shvorak — CHANGES_REQUESTED** [review]
Please check the comments

**Review by VadimMentorQA — CHANGES_REQUESTED** [review]
Check comments.
Revise all files added to your PR - the task requires `src/getCoinCombination.test.js` only

**Review by VadimMentorQA — CHANGES_REQUESTED** [review]
3. Not fixed. Your test titles must reflect the goal of the test. Add proper naming for these tests, and rewrite every your test title as a test case summary. And 'correct' or 'invalid' we don't include in test case summaries.

**Review by VadimMentorQA — CHANGES_REQUESTED** [review]
It would be best if you changed `src/getCoinCombination.test.js` only.  Remove all other files from your pull request, please.
Could you make sure your test names have equivalence class names?

File: src/getCoinCombination.test.js (hunk 2 above)
Comment (Oznobishina) [reply src/getCoinCombination.test.js]: Я не вижу новых замечаний, что нужно исправить?

**Review by VadimMentorQA — CHANGES_REQUESTED** [review]
What exactly was changed?
You still have 15 changed files - we don't need it in the pull request, and we don't need these changes in our master. 
Clean up your PR, please. If you have any questions - feel free to ask in #qa_chat
